docs/tutorial/accelerator_design.md

```diff
@@ -1,25 +1,71 @@
 # Accelerator Design
 
-## TODOs:
-- Describe the accelerator of interest
-- Show a detailed block diagram of the signals
-- Mention and describe what the CSR Manager does
-- Describe the CSRs to be used
-    - Mode: +, -, x, XOR
-    - Data length
-    - Start
-    - Busy
-    - Counter
-- Have a timing diagram interface to describe what happens for the signals
-- Indicate which repository to see this structure
+Let's first dive into the SNAX shell which is the encapsulated yellow box from the main figure in [Architectural Overview](./architectural_overview.md). The figure below is the same shell but with more signal details:
 
-## Accelerator Architecture
 
-### Modes and Features
+We labeled a few important details about the shell:
 
-### Interfaces
+1 - The entire SNAX accelerator wrapper encapsulates the streamers, the CSR manager, and the accelerator data path.
 
-### CSR Manager
+2 - The CSR manager handles the CSR read and write transactions between the Snitch core and the accelerator data path. Towards the Snitch core side, CSR transactions are handled with decoupled interfaces (`csr_req_*` and `csr_rsp_*`) of requests and responses. Towards the accelerator side, the read-write registers (`register_rw_set`) use a decoupled interface (`register_rw_valid` and `register_rw_ready`) while the read-only registers (`register_ro_set`) use a direct mapping. There are more details in [CSR Manager Design](./csrman_design.md).
 
-## Timing Diagrams
+3 - The streamer provides flexible data access from the L1 TCDM to the accelerator. It serves as an intermediate interface between the TCDM interconnect and the accelerator. On the interconnect side the streamer controls the TCDM request and response (`tcdm_req` and `tcdm_rsp`) interfaces towards the memory. On the accelerator side, the streamer has its own data decoupled interfaces (`acc2stream_*` and `stream2acc_*`). The direction from the accelerator to the streamer is write-only ports while the direction from the streamer to the accelerator is read-only ports. More details are in [Streamer Design](./streamer_design.md).
+
+4 - The accelerator data path is the focus of this section. In our example, we will use a very simple ALU datapath with basic operations only. The SNAX ALU is already built for you. You can find it under the `./hw/snax_alu/` directory. Take time to check the simple design.
+
+
+# SNAX ALU Datapath
+
+{%
+   include-markdown '../../hw/snax_alu/doc/snax_alu.md'
+   start="# SNAX ALU Accelerator Datapath"
+   comments=false
+%}
+
+# Adding Your Accelerator to the Configuration File
+
+You can add your accelerator configurations in a configuration file. The SNAX ALU configuration `snax-alu.hjson` has core templates which configure the Snitch core and how it connects to an accelerator. The `snax-alu` core template is:
+
+```json
```

Comment:
Note that this is an `hjson` file, it is actually invalid as a `json` file
```suggestion
```hjson
```

Reply:
Okie! I only put json only because hjson does not seem to highlight appropriately, but okay! hjson it is to be consistent!

Reply:
Yeah that's what I thought. I'm afraid github does not render hjson (yet?, if ever?). The problem is that your json has comments and that is not allowed, so if it will render as json it might render incorrectly as well.